cloud/gcp/modules/compute/main.tf

```diff
@@ -0,0 +1,69 @@
+resource "google_cloud_run_service" "civiform_application_run_service" {
+  name     = "cloudrun-civiform"
+  location = var.region
+
+  ##TODO(@ktoor): Change to correct docker image. Will need to push the image to gcr through some mechanism.
+  template {
+    spec {
+      containers {
+        image = "us-docker.pkg.dev/cloudrun/container/hello"
```

Comment:
suggestion (blocking): actual image is `docker.io/civiform/civiform:latest`

Reply:
Added it in the current workspace I have. This was for testing purposes. Cloud run cannot directly refer to docker.io repo. Has to be uploaded to GCR

Reply:
Ah that's good to know, thanks